bin/gen-pgbouncer-conf.sh

```diff
@@ -67,22 +80,54 @@ tcp_keepidle = ${PGBOUNCER_TCP_KEEPIDLE:-7200}
 tcp_keepintvl = ${PGBOUNCER_TCP_KEEPINTVL:-75}
 EOFEOF
 
+# add certificates, key to pgbouncer.ini and change sslmode 
+# if all of certs and key are present and also adjust the certificates' formate 
+# from outputs of envrionment variables
+#
+if [ -n "${CLIENT_TLS_KEY_FILE}" ] && [ -n "${CLIENT_TLS_CRT_FILE}" ] && [ -n "${CLIENT_TLS_CA_FILE}" ]
+then 
+  echo -e "-----BEGIN CERTIFICATE-----" > ${PGBOUNCER_DIR}/pgbouncer.crt
```

Comment:
you appear to be reconstructing a certificate file based on a single line string, why not just use something like
```
ENCRYPTED=$(cat config/dumb-ssl/demo.test.crt| kubectl --context aws-integration-theta-purple exec -i web-0 -- ali bin/enigma.rb --encrypt)
```
after decrypting from SEO, you can just do
```
echo "${ENCRYPTED_CERT}" > /path/to/cert
```

Reply:
Thanks! I had tried same way as "echo ${ENCRYPTED_CERT}" > /path/to/cert".  but in here "${CLIENT_TLS_KEY_FILE}" is from "$INTERNAL_TLS_KEY" environment variable. the content is 1 line (somehow the '\n' was replaced by ' ', I had not expected this and spend a few hours on it), so had to do this reconstruction to get it back to right format).

Reply:
note the quotes:
```
echo "${ENCRYPTED_CERT}" > /path/to/cert
```
I just validated with the SEO var `TEST_CRT` on theta, doing `echo` directly seems to work fine.